components/layout_2020/dom_traversal.rs

```diff
@@ -448,10 +459,31 @@ where
                             vec.push(PseudoElementContentItem::Replaced(replaced_content));
                         }
                     },
+                    ContentItem::OpenQuote | ContentItem::CloseQuote => {
+                        // TODO(xiaochengh): calculate quote depth
+                        let depth: usize = 0;
+                        let quote: Option<String> = match &pseudo_element_style.get_list().quotes {
+                            Quotes::QuoteList(quote_list) => {
+                                quote_list.0.get(depth).map(|quote_pair| get_quote_from_pair(
+                                        item,
+                                        // TODO: use `as_str()` once it is stable.
+                                        // See https://github.com/rust-lang/rust/issues/130366
+                                        quote_pair.opening.split_at(0).1,
+                                        quote_pair.closing.split_at(0).1,
+                                    ))
+                            },
+                            Quotes::Auto => {
+                                let lang = &pseudo_element_style.get_font()._x_lang;
+                                let quotes = crate::quotes::quotes_for_lang(lang.0.as_ref(), depth);
```

Comment:
I would add `use crate::quotes::quotes_for_lang;` at the top of the file and just `quotes_for_lang` here.

Reply:
Done.